libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -10,6 +10,7 @@
 #include <AP_BattMonitor/AP_BattMonitor.h>
 #include <AP_AHRS/AP_AHRS.h>
 #include <AP_Arming/AP_Arming.h>
+#include <AP_Vehicle/AP_Vehicle.h>
```

Comment:
The one thing I am unsure about is the circular includes. 

Oddly enough, AP_Vehicle.h doesn't include AP_DDS_Client.h.

Reply:
`AP_Vehicle.h` forward declares `AP_DDS_Client`, has a pointer member variable `dds_client`, and pulls the include into the translation unit. So I think that's ok.